build/utils/dotnet/solution/solution.go

```diff
@@ -207,12 +207,12 @@ func (solution *solution) loadSingleProject(project project.Project, log utils.L
 	// First we wil find the project's dependencies source.
 	// It can be located directly in the project's root directory or in a directory with the project name under the solution root
 	// or under obj directory (in case of assets.json file)
-	projectRootPath := project.RootPath()
-	projectPathPattern := filepath.Join(projectRootPath, dependencies.AssetDirName) + string(filepath.Separator)
-	projectNamePattern := string(filepath.Separator) + project.Name() + string(filepath.Separator)
+	projectRootPath := strings.ToLower(project.RootPath())
+	projectPathPattern := strings.ToLower(filepath.Join(projectRootPath, dependencies.AssetDirName) + string(filepath.Separator))
+	projectNamePattern := strings.ToLower(string(filepath.Separator) + project.Name() + string(filepath.Separator))
```

Comment:
Question - doesn't the fact that we now use `strings.ToLower` when building the path (in multiple places in this file) can make the path not be accessible on some platforms, which use case-sensitivity for the files and directory names?